server/src/main/java/fr/uga/l3miage/tp4/models/EcosSessionProgrammationStepEntity.java

```diff
@@ -0,0 +1,19 @@
+package fr.uga.l3miage.tp4.models;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+public class EcosSessionProgrammationStepEntity {
+    @Id
+    private Long id;
+    private final LocalDateTime dateTime = LocalDateTime.now();
```

Comment:
le final ne voulais pas dire ça, il voulais dire que tu ne peux pas le mettre à jour donc 
```suggestion
    @Column(updatable = false)
    private LocalDateTime dateTime;
```